File: Arrays/Second Largest/largest.c
int getSecondLargest(int *arr, int n) {
    // code here
    int lar = -1;
    for(int i=0;i<n;i++)
    {
        if(arr[i]>lar)
            lar = arr[i];
    }
    
    int second = -1;
    for(int i=0;i<n;i++)
    {
        if(arr[i]>second && arr[i]<lar)
            second = arr[i];
    }
    
    return second;
}

File: Arrays/Second Largest/largest.py
class Solution:
    def getSecondLargest(self, arr):
        # Code Here
        lar = max(arr)
        second = -1
        n = len(arr)
        for i in range(n):
            if(arr[i]>second and arr[i]<lar):
                second = arr[i]
                
        return second
